invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/CommunitiesList.js

```diff
@@ -0,0 +1,116 @@
+// This file is part of InvenioRDM
+// Copyright (C) 2023 CERN.
+//
+// InvenioRDM is free software; you can redistribute it and/or modify it
+// under the terms of the MIT License; see LICENSE file for more details.
+
+import { i18next } from "@translations/invenio_app_rdm/i18next";
+import React, { Component } from "react";
+// import PropTypes from "prop-types";
+import axios from "axios";
+import { withCancel } from "react-invenio-forms";
+import { Container, Item, Button, Placeholder, Message } from "semantic-ui-react";
+import { Image } from "react-invenio-forms";
+import { CommunitiesListModal } from "./CommunitiesListModal/CommunitiesListModal";
+
+export class CommunitiesList extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      loading: true,
+      communities: null,
+      error: null,
+    };
+  }
+
+  componentDidMount() {
+    this.getCommunities();
+  }
+
+  componentWillUnmount() {
+    this.cancellableFetchCommunities?.cancel();
+  }
+
+  getCommunities = async () => {
+    this.cancellableFetchCommunities = withCancel(this.fetchRecordCommunities());
+    this.setState({
+      loading: true,
+      error: null,
+    });
+    try {
+      const res = await this.cancellableFetchCommunities.promise;
+      this.setState({
+        communities: res.data?.hits?.hits,
+        loading: false,
+      });
+    } catch (error) {
+      if (error !== "UNMOUNTED") {
+        this.setState({
+          loading: false,
+          error: i18next.t("An error occurred while fetching communities."),
+        });
+      }
+    }
+  };
+
+  fetchRecordCommunities = async () => {
+    // TODO: Update endpoint to be record specific
+    return await axios.get("/api/communities", {
```

Comment:
temporarily, you could do (to already have the correct communities for testing):
```suggestion
    return await axios.get("/api/communities?q=id:${community.id} OR id:${community.id} OR ...", {
```
where `q=id:${community.id} OR id:${community.id}` this part could be generated base on communities included in the record

Reply:
But it is not the record id we want to include? I.e. something like `/api/record/{id}/communities` ?

Reply:
If it's only for testing, is it necessary to change?

Reply:
> But it is not the record id we want to include in the end? I.e. something like `/api/records/{id}/communities` ?

yes this is how it will look like at the end, but until then you could search in communities endpoint. Either way it was just a suggestion how to already display correct data before the endpoint is completed